src/controllers/check.your.answers.controller.ts

```diff
@@ -26,12 +27,16 @@ const route = async (req: Request, res: Response, next: NextFunction): Promise<v
       const request: IExtensionRequest = sessionService.getRequest(req.chSession);
       const fullRequest: ExtensionFullRequest =
         await apiClient.getFullRequest(companyNumber, token, request.extension_request_id);
+      let companySummaryListRows = buildCompanySummaryListRows(companyInSession, true);
+      companySummaryListRows = companySummaryListRows.concat(
```

Comment:
Just wondering if you could encapsulate the logic that adds the e-mail JSON into the 'buildCompanySummaryListRows()' function? Pass in another optional 'email' parameter which is checked to see if JSON is added....